diff -Nur xpcd-2.08.orig/libpcd/Makefile.in xpcd-2.08/libpcd/Makefile.in
--- xpcd-2.08.orig/libpcd/Makefile.in	Sat Oct 10 22:39:47 1998
+++ xpcd-2.08/libpcd/Makefile.in	Fri Jan 26 01:12:08 2001
@@ -17,19 +17,20 @@
 	$(srcdir)/huff.c \
 	$(srcdir)/inter.c
 
-OBJS=file.o yuv2rgb.o huff.o inter.o
+OBJS=file.lo yuv2rgb.lo huff.lo inter.lo
+
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $<
 
 all: libpcd.a
 
 libpcd.a: $(OBJS)
-	rm -f $@
-	ar rc $@ $(OBJS)
-	ranlib $@
+	libtool $(CC) $(LDFLAGS) -o libpcd.la $(OBJS) -rpath @libdir@
 
 install-lib:
 	$(MKDIR) $(libdir)
 	$(MKDIR) $(incdir)
-	$(INSTALL_DATA) libpcd.a $(libdir)
+	libtool $(INSTALL_DATA) libpcd.la $(libdir)
 	$(INSTALL_DATA) $(srcdir)/pcd.h $(incdir)
 
 install:
@@ -38,10 +39,10 @@
 	$(DEPEND) -- $(CFLAGS) -- $(SRCS)
 
 clean:
-	-rm -f *~ *.bak *.o
+	-rm -f *~ *.bak *.o *.lo
 
 realclean:
-	-rm -f libpcd.a Makefile
+	-rm -f libpcd.a .libs *.la *.so Makefile
 
 #-------------------------------------------------------------------------
 # DO NOT DELETE THIS LINE -- make depend depends on it.
diff -Nur xpcd-2.08.orig/test/Makefile.in xpcd-2.08/test/Makefile.in
--- xpcd-2.08.orig/test/Makefile.in	Sun Oct 17 13:09:03 1999
+++ xpcd-2.08/test/Makefile.in	Fri Jan 26 01:12:50 2001
@@ -4,7 +4,7 @@
 include ../Make.config
 
 CFLAGS=@CFLAGS@ -Wall -I.. -I$(srcdir)/../libpcd @X_CFLAGS@
-LDLIBS=-L../libpcd -lpcd
+LDLIBS=-L../libpcd/.libs -lpcd
 
 PROGS=@TEST_PROGS@
 
diff -Nur xpcd-2.08.orig/xpcd/Makefile.in xpcd-2.08/xpcd/Makefile.in
--- xpcd-2.08.orig/xpcd/Makefile.in	Sun Oct 11 20:47:50 1998
+++ xpcd-2.08/xpcd/Makefile.in	Fri Jan 26 01:13:06 2001
@@ -6,7 +6,7 @@
 
 CFLAGS=@CFLAGS@ -Wall -I.. -I$(srcdir)/../libpcd @X_CFLAGS@ \
 	-DSYSRC="\"$(x11_libdir)/xpcd/system.xpcdrc\""
-LDLIBS=-L../libpcd -lpcd -lm \
+LDLIBS=-L../libpcd/.libs -lpcd -lm \
 	@X_LIBS@ @X_TOOLKIT@ -lXt @X_PRE_LIBS@ @MIT_SHMEM@ \
 	-lXmu -lX11 @X_EXTRA_LIBS@
 
