--- xpcd-2.08/xpcd/cfgfile.c.orig	1997-09-12 11:38:46.000000000 +0200
+++ xpcd-2.08/xpcd/cfgfile.c	2003-08-18 22:50:19.126317960 +0200
@@ -18,9 +18,10 @@
     int            *intval;
     char          **strval;
 
-    strcpy(fn, getenv("HOME"));
-    strcat(fn, "/");
-    strcat(fn, own);
+    if (getenv("HOME"))
+	snprintf(fn, sizeof(fn) - 1, "%s/%s", getenv("HOME"), own);
+    else
+	strcpy(fn, own);
     if (NULL == (fp = fopen(fn, "r"))) {
 	strcpy(fn, all);
 	if (NULL == (fp = fopen(all, "r")))
