diff -Nur xpcd-2.08.orig/configure.in xpcd-2.08/configure.in
--- xpcd-2.08.orig/configure.in	Sun Oct 17 13:27:25 1999
+++ xpcd-2.08/configure.in	Fri Jan  5 13:00:21 2001
@@ -89,12 +89,20 @@
 dnl    TEST_PROGS="$TEST_PROGS plug-0.54"
 dnl  fi
 dnl  echo -n "trying gimp 0.99: "
-  AC_CHECK_LIB(gimp, gimp_image_new,,,-lglib)
+  AC_MSG_CHECKING([for gimp_image_new in -lgimp])
+  LIBS="-lgimp -lglib -L/usr/X11R6/lib"
+  AC_TRY_LINK([char PLUG_IN_INFO;
+char gimp_image_new();],[gimp_image_new()],
+AC_MSG_RESULT(yes)
+LIBS=-lgimp
+AC_DEFINE_UNQUOTED(HAVE_LIBGIMP),
+AC_MSG_RESULT(no)
+LIBS="")
   if test "$LIBS" != ""; then
     LIBGIMP99="$LIBS -lglib"; LIBS=""
     XPCD_PROGS="$XPCD_PROGS xpcd-gate";
     GIMP_PLUGIN_DIR=""
-    for dir in /usr/lib/gimp/*/plug-ins /usr/local/lib/gimp/*/plug-ins; do
+    for dir in /usr/lib/gimp/*/plug-ins /usr/local/lib/gimp/*/plug-ins /usr/X11R6/lib/gimp/*/plug-ins; do
       if test -d $dir; then
 	GIMP_PLUGIN_DIR="$dir"
 	break
